invenio_oauthclient/contrib/keycloak/helpers.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 FAIR Data Austria.
+#
+# Invenio-Keycloak is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Helper functions for the endpoint handlers."""
+
+import jwt
+from flask import current_app
+
+
+def _get_config_item(item):
```

Comment:
Some of these configs are probably required, in this case I would require them and not use the `dict.get()`.